### ISSUE TITLE
[Bug] - Notes cannot be shown

### ISSUE THREAD

**Issue by clone1612** [BoostIO/Boostnote#637]
Version: 0.8.10
Operating System: Linux (Arch x64)

Trying to use the latest version notes just will not show. When I select a note this view remains (the new post screen):
![deepinscreenshot20170612191915](https://user-images.githubusercontent.com/4518410/27046343-568e0a3e-4fa4-11e7-8ea7-0e27da77ba3d.png)

The console is showing following error:
![deepinscreenshot20170612191740](https://user-images.githubusercontent.com/4518410/27046364-6a7527b2-4fa4-11e7-9871-00db37d5e730.png)

Following that link shows the exact contents of the error:
![deepinscreenshot20170612192237](https://user-images.githubusercontent.com/4518410/27046398-8e2eb948-4fa4-11e7-8533-665aa48b1ef8.png)

**Comment by asmsuechan** [comment]
Hi, @clone1612. Could you paste the note as long as you can? The note is probably located in `~/Boostnote/notes/*.cson`.

**Comment by clone1612** [comment]
This is the full content of the note:
```
type: "MARKDOWN_NOTE"
folder: "95437e1f8b7bd1705f81"
title: "Chapter 5"
content: '''
  # Chapter 5
  What is the pattern of Model-Driven Architectures (MDA)?
  - Design a Platform Independent Model (PIM)
  - Automatically transform to a Platform Specific Model (PSM)
  - Repeat until you reach code
  
  Benefits?
  - __Productivity__ with a shift of focus from PSM -> PIM and reusing the PIM->PSM transformations
  - __Portability__ since PIMs are portable to multiple platforms
  - __Maintenance and Documentation__ because automatic transformations keep (derived) artefacts in sync
  
  All models predefined by MDA:
  - __CIM:__ Computation Independent Model which is a conceptual data-centric model of the system's _application domain_
  - __PIM__
  - __PSM__
  - __PM:__ Platform Model describing platform features that we can interface with
'''
tags: []
isStarred: false
createdAt: "2017-06-11T16:28:39.518Z"
updatedAt: "2017-06-11T16:34:59.547Z"
```
Happens with all notes and I can't seem to figure out why. They worked fine with 0.8.9 yesterday but with the new version none can be shown, all showing the same error.

**Comment by asmsuechan** [comment]
Umm, actually, we're working for release the version **now**. It means v0.8.10 should not be released yet :fearful:

**Comment by clone1612** [comment]
Ow well that explains a lot :) But maybe change the release page then, thanks for the very quick response!
![deepinscreenshot20170612193545](https://user-images.githubusercontent.com/4518410/27046899-7151cb42-4fa6-11e7-9f37-ea90aa82b899.png)

**Comment by clone1612** [comment]
More specific error + a new one which pops up after switching between notes:
![deepinscreenshot20170612202558](https://user-images.githubusercontent.com/4518410/27048798-777eb398-4fad-11e7-9f91-c89f401016aa.png)
So to summarize my problem:
- Notes can not be shown, screen remains on create a new post
- Clicking on different notes leads to the errors shown in the previous image
- Not happening in 0.8.9 (same note storage, did not add anything new with 0.8.10)
- Happens in 0.8.10 on Arch Linux (x64)

**Comment by clone1612** [comment]
Continuing to identify where the root of the problem is I finally found an actual helpful error which might give you an idea of where the issue can be found:
![deepinscreenshot20170614113604](https://user-images.githubusercontent.com/4518410/27125786-b01af970-50f5-11e7-9771-8c76287da41f.png)
Which leads to a real error message pointing to an actual function:
![deepinscreenshot20170614113350](https://user-images.githubusercontent.com/4518410/27125807-c4842e36-50f5-11e7-9cfe-b18609112604.png)
So something related to the **MarkdownNoteDetail** rendering method passing a wrong element type or so.

**Comment by mathewmeconry** [comment]
:+1:

**Comment by kazup01** [comment]
Probably improved in v0.8.11. 
Is this bug still occurring?

**Comment by clone1612** [comment]
Fixed with v0.8.11, thanks!

**Comment by kazup01** [comment]
Cool! Thanks for waiting!
